From 8c2fb922b96e9506631ef68dec89ba7e5e0f1ca4 Mon Sep 17 00:00:00 2001
From: Marcel Telka <marcel@telka.sk>
Date: Wed, 25 Apr 2018 00:28:01 +0200
Subject: [PATCH] 9447 NFS unmount is slow Reviewed by: Toomas Soome
 <tsoome@me.com> Reviewed by: Arne Jansen <arne@die-jansens.de> Reviewed by:
 Ken Mays <kmays2000@gmail.com> Reviewed by: Evan Layton
 <evan.layton@nexenta.com> Approved by: Dan McDonald <danmcd@joyent.com>

---
 usr/src/uts/common/fs/nfs/nfs_client.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/usr/src/uts/common/fs/nfs/nfs_client.c b/usr/src/uts/common/fs/nfs/nfs_client.c
index 995b145b2b..719fc04db2 100644
--- a/usr/src/uts/common/fs/nfs/nfs_client.c
+++ b/usr/src/uts/common/fs/nfs/nfs_client.c
@@ -1362,8 +1362,8 @@ nfs_async_manager_stop(vfs_t *vfsp)
 
 int
 nfs_async_readahead(vnode_t *vp, u_offset_t blkoff, caddr_t addr,
-    struct seg *seg, cred_t *cr, void (*readahead)(vnode_t *, u_offset_t,
-    caddr_t, struct seg *, cred_t *))
+    struct seg *seg, cred_t *cr, void (*readahead)(vnode_t *,
+    u_offset_t, caddr_t, struct seg *, cred_t *))
 {
 	rnode_t *rp;
 	mntinfo_t *mi;
@@ -1462,8 +1462,8 @@ noasync:
 
 int
 nfs_async_putapage(vnode_t *vp, page_t *pp, u_offset_t off, size_t len,
-    int flags, cred_t *cr, int (*putapage)(vnode_t *, page_t *, u_offset_t,
-    size_t, int, cred_t *))
+    int flags, cred_t *cr, int (*putapage)(vnode_t *, page_t *,
+    u_offset_t, size_t, int, cred_t *))
 {
 	rnode_t *rp;
 	mntinfo_t *mi;
-- 
2.17.2 (Apple Git-113)

